src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java

```diff
@@ -1,16 +1,30 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import java.util.Optional;
+import mate.academy.hibernate.relations.dao.impl.ActorDaoImpl;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
+import org.hibernate.SessionFactory;
 
 public class ActorServiceImpl implements ActorService {
+    private SessionFactory factory;
+
+    public ActorServiceImpl(SessionFactory factory) {
+        this.factory = factory;
+    }
+
     @Override
     public Actor add(Actor actor) {
-        return null;
+        return new ActorDaoImpl(factory).add(actor);
```

Comment:
do not create new ActorDaoImpl

Reply:
Unfortunately, I don't understand what exactly is wrong. In the past work, it was through Inject. Nothing about that happened in this work. Bring the Dao field into a variable class?